src/test/java/com/google/api/codegen/testdata/simplecompute_gapic.yaml

```diff
@@ -242,6 +242,8 @@ interfaces:
   collections:
   - name_pattern: projects/{project}/dummyObjects/{dummyObject}
     entity_name: dummyObject
+  - name_pattern: projects/{project}/dummyObjects/{resource}
```

Comment:
Changes in this file are also just a direct, autogenerated consequence of the changes in compute.v1.json.